src/main/java/saps/dispatcher/core/SubmissionDispatcher.java

```diff
@@ -18,6 +18,7 @@
 
 public class SubmissionDispatcher {
 
+    public static final String EXECUTION_TAGS_FILE_PATH_KEY = "EXECUTION_SCRIPT_TAGS_FILE_PATH";
```

Comment:
Why `_KEY`?

Reply:
Because it is the key to a system variable.